data/sql/updates/pending_db_world/rev_1718702527397206595.sql

```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `command` SET `help` = REPLACE(`help`, 'trigered', 'triggered') WHERE `name` like 'cast%';
```

Comment:
I would prefer you to update the entire text instead of using Replace.

Reply:
caught another typo doing it with inserts :laughing: